src/thok.rs

```diff
@@ -141,6 +171,8 @@ impl Thok {
             self.wpm = 0.0;
         }
         self.accuracy = ((correct_chars.len() as f64 / self.input.len() as f64) * 100.0).round();
+
+        let _ = self.save_results();
```

Comment:
Any reason for the `let _`?

Reply:
It's the least gross way to ignore an error. It could be replaced with `.ok()` or `.drop()`.

I think it makes sense to ignore the error here, as failing to log results shouldn't crash the program. Maybe instead it should log to stderr or display on the results screen?